x/pairing/client/cli/tx_stake_provider.go

```diff
@@ -188,6 +190,12 @@ func CmdBulkStakeProvider() *cobra.Command {
 						continue
 					}
 
+					q := spectypes.NewQueryClient(clientCtx)
```

Comment:
rule of thumb is to not create a query when you do a transaction.
let it fail on the node side